rules/windows/builtin/win_invoke_obfuscation_via_enc_obfuscation_8_services.yml

```diff
@@ -0,0 +1,42 @@
+action: global
+title: Invoke-Obfuscation ENCODING OBFUSCATION\8
+id: 00559152-8ef6-4e83-8daf-69ce2c98c850
+description: Detects Obfuscated Powershell via ENCODING OBFUSCATION\8
+status: experimental
+author: Timur Zinniatullin, oscd.community
+date: 2020/10/18
+references:
+    - https://github.com/Neo23x0/sigma/issues/1009 #(Task 18)
+tags:
+    - attack.defense_evasion
+    - attack.t1027
+    - attack.execution
+    - attack.t1059.001
+falsepositives:
+    - Unknown
+level: medium
+detection:
+    selection_1:
+        - ImagePath|re: '(?i)\'\s+\'\s?\|(?:foreach|\s?%).*(?:\[string\]::join|remove\.tostring).*trim.*split.*(?:char|substring\.tostring).*\}'
```

Comment:
maybe this construction is going to fix that?

```
        ImagePath|re: >
          '(?i)\'\s+\'\s?\|(?:foreach|\s?%).*(?:\[string\]::join|remove\.tostring).*trim.*split.*(?:char|substring\.tostring).*\}'
```

It works locally. Here is the result:

```
(EventID:"7045" AND ImagePath:/'(?i)\'\s+\'\s?\|(?:foreach|\s?%).*(?:\[string\]::join|remove\.tostring).*trim.*split.*(?:char|substring\.tostring).*\}'
/)
(EventLog:"Microsoft\-Windows\-Sysmon" AND EventID:"6" AND ImagePath:/'(?i)\'\s+\'\s?\|(?:foreach|\s?%).*(?:\[string\]::join|remove\.tostring).*trim.*split.*(?:char|substring\.tostring).*\}'
/)
(EventLog:"Security" AND EventID:"4697" AND ImagePath:/'(?i)\'\s+\'\s?\|(?:foreach|\s?%).*(?:\[string\]::join|remove\.tostring).*trim.*split.*(?:char|substring\.tostring).*\}'
/)

```

Reply:
Had to add `-` to the `ImagePath|re: >` to strip the line. my fault. I've just went through all your rules and updated them